src/locales/en.json

```diff
@@ -31,7 +31,13 @@
       "their_mask_header": "Their mask",
       "filtered_ensure_airflow": "Ensure sufficient flow through the filter for your room. For help on how to do this, see <1>here</1>.",
       "volume_header": "Volume",
-      "volume_question": "How much is everyone talking?"
+      "volume_question": "How much is everyone talking?",
+      "your_vaccine_header": "Your Vaccine",
+      "your_vaccine_label": "Which vaccine did you receive?",
+      "your_vaccine_doses_label": "How many dose have you received?",
+      "no_vaccine": "No vaccine",
+      "doses_7_days_ago": "Only count a dose if you received it at least 7 days ago.",
```

Comment:
"If your last dose was <1 week ago, subtract one dose when answering--the last dose may not be protecting you yet."

is too long, but addresses the following:

1. With the current version, I have to read the entire line before I can decide if it applies to me
2. If I match the condition I pattern-matched from this instruction (at least 7 days ago), I need to do nothing. In fact, if I was last shot *at most* 7 days ago, I need to take extra action. This is an opportunity for error.
2. If I received my last dose 3 days ago and read the entire line and decided it applied to me, I have to then sort out what it means to count a dose and what it means to *not* count a dose. With some thought, I might come to the conclusion that I need to subtract one from the number I started with and then separately perform the subtraction, or do this: "I had two shots. Does the first one count? Yes. Does the second one count? No." This is more effort than strictly necessary.

Reply:
Done.

I honestly don't care that much about errors here. It'll only affect people for a week, and the actual # of days you need to wait is actually fuzzy; they had to choose an endpoint before starting the studies. Plausibly you're protected by the time your fever from the second dose stops. I'm not terribly worried about people with fevers, chills, and aches from the second dose getting into trouble, and the multiplier from the first dose is so mild that I don't think anyone will get into much trouble. Erroneously doubling your budget for a week is negligible on a year scale.